wiring/src/spark_wiring_string.cpp

```diff
@@ -153,6 +154,22 @@ String::String(unsigned long value, unsigned char base)
 	*this = buf;
 }
 
+String::String(long long value, unsigned char base)
+{
+	init();
+	char buf[66] {};  // Assuming a maximum of 64-bit binary representation + '-' + null terminator.
```

Comment:
Let's make this a constant. This should also be 24 for worst case scenario for 64-bit ints (including hex/octal and other cases).

Style: Use `buf[24] = {}` initialization. Same in other places.

Reply:
done